Common/GPU/Vulkan/VulkanMemory.h

```diff
@@ -134,3 +136,30 @@ class VulkanPushBuffer {
 	uint8_t *writePtr_ = nullptr;
 	VkBufferUsageFlags usage_;
 };
+
+class VulkanDescSetPool {
```

Comment:
Should document with a comment that descriptors returned from this are only valid for the current frame (since a resize queues a destruction of the previous pool, with no in-use checks - which is fine, but only for this use case).

Reply:
Added a comment.  It can in theory live past that frame as long as `clear_()` safely purges the cache, but indeed the intent is only in-frame usage.  I think one of the users does decimate/Reset on prime frame counts, not every frame, so if it didn't grow it could last several frames.

-[Unknown]